drivers/si70xx/include/si70xx_internals.h

```diff
@@ -55,15 +55,25 @@ extern "C" {
  * @{
  */
 #if defined(MODULE_SI7006)
-#define SI70XX_ID                   (0x06)
+#define SI70XX_ID                   (6)
 #elif defined(MODULE_SI7013)
-#define SI70XX_ID                   (0x0D)
+#define SI70XX_ID                   (13)
 #elif defined(MODULE_SI7020)
-#define SI70XX_ID                   (0x14)
+#define SI70XX_ID                   (20)
 #elif defined(MODULE_SI7021)
-#define SI70XX_ID                   (0x15)
+#define SI70XX_ID                   (21)
+#elif defined(MODULE_SI7050)
+#define SI70XX_ID                   (50)
+#elif defined(MODULE_SI7051)
+#define SI70XX_ID                   (51)
+#elif defined(MODULE_SI7053)
+#define SI70XX_ID                   (53)
+#elif defined(MODULE_SI7054)
+#define SI70XX_ID                   (54)
+#elif defined(MODULE_SI7055)
+#define SI70XX_ID                   (55)
```

Comment:
Changed all these values to decimal because they make a lot more sense in decimal (is the part number).

Reply:
I agree.